corehq/apps/locations/models.py

```diff
@@ -228,7 +228,7 @@ class SQLLocation(SyncSQLToCouchMixin, MPTTModel):
     location_type = models.ForeignKey(LocationType)
     site_code = models.CharField(max_length=255)
     external_id = models.CharField(max_length=255, null=True)
-    metadata = json_field.JSONField(default={})
+    metadata = jsonfield.JSONField(default={})
```

Comment:
Sure but `{}` is a mutable object which makes it a bad candidate for
function args defaults